.github/workflows/run-tests.yml

```diff
@@ -7,18 +7,23 @@ on:
     branches: [main, 1.x, 2.x]
 
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
+  build:
+    runs-on: ubuntu-latest
```

Comment:
why the rename of the job?
and why the removal of Windows from the configuration? We still want to support Windows and we want to test against it.

Reply:
I ended up uploading this to test the actions. Revert